litestar/config/compression.py

```diff
@@ -48,16 +56,26 @@ class CompressionConfig:
     """A pattern or list of patterns to skip in the compression middleware."""
     exclude_opt_key: str | None = None
     """An identifier to use on routes to disable compression for a particular route."""
+    compression_facade: type[CompressionFacade] = GzipCompression
+    """The compression facade to use for the actual compression."""
+    backend_config: Any = None
```

Comment:
Ideally, this should be used for both `gzip` and `brotli` specific configurations instead of it polluting `CompressionConfig.` However, I wasn't able to find a clean way to do that without breaking backwards compatibility.